pom.xml

```diff
@@ -75,7 +75,7 @@
       <dependency>
         <groupId>io.jenkins.tools.bom</groupId>
         <artifactId>bom-2.319.x</artifactId>
-        <version>1090.v0a_33df40457a_</version>
+        <version>1117.v62a_f6a_01de98</version>
```

Comment:
picks up an updated git plugin (4.10.3) that does not use local git templates when init'ing a repo
jenkinsci/git-plugin#1207